kili/services/__init__.py

```diff
@@ -2,8 +2,9 @@
 Python SDK service layer
 """
 
+from kili.services.asset_download import download_asset_media
```

Comment:
I like the get_asset service idea because we will implement other processing later. For example the get_assets features in autoML should be integrated into the SDK

Reply:
👍